ouroboros-consensus/src/Ouroboros/Consensus/Demo/Byron/Elaborate.hs

```diff
@@ -45,9 +44,8 @@ elaborateTx :: HasCallStack
             => NodeConfig ByronEBBExtNodeConfig
             -> Mock.Tx -> GenTx (ByronBlockOrEBB cfg)
 elaborateTx (WithEBBNodeConfig cfg) (Mock.Tx ins outs) =
-    mkByronTx $ CC.UTxO.ATxAux (annotate tx) (annotate witness)
+    mkByronTx $ CC.UTxO.annotateTxAux $ CC.UTxO.mkTxAux tx witness
```

Comment:
This is better than how I did it in https://github.com/input-output-hk/ouroboros-network/pull/999/files?file-filters%5B%5D=.hs#diff-347afb0d50648cb62427905c952ee015R50